frontend/sw360-portlet/src/main/java/org/eclipse/sw360/portal/common/AttachmentPortletUtils.java

```diff
@@ -147,7 +152,9 @@ private void serveAttachmentBundle(List<AttachmentContent> attachments, Resource
             Optional<Object> context = getContextFromRequest(request, user);
 
             if(context.isPresent()){
-                try (InputStream attachmentStream = getStreamToServeAFile(attachments, user, context.get())) {
+                try (InputStream attachmentStream = (downloadAllAttachments
+                        ? getStreamToServeBundle(attachments, user, context.get())
+                        : getStreamToServeAFile(attachments, user, context.get()))) {
```

Comment:
The logic should work here (if I am correct), but it still a bit confusing with the method names since "...AFile()" also serves a bundle if there is more than one attachment in the list. Maybe you can spend five minutes thinking if you can improve something just regarding method names or even in method implementation to make it more straight forward. But if you do not find a quick solution, this is okay.